tests/test_repository_tool.py

```diff
@@ -1233,25 +1257,25 @@ def test_remove_target(self):
     self.assertEqual(self.targets_object.target_files, {})
 
     # Add a target so that remove_target() has something to remove.
-    target_filepath = os.path.join(self.targets_directory, 'file1.txt')
+    target_filepath = 'file1.txt'
     self.targets_object.add_target(target_filepath)
 
     # Test remove_target()'s behavior.
-    self.targets_object.remove_target(os.path.basename(target_filepath))
+    self.targets_object.remove_target(target_filepath)
     self.assertEqual(self.targets_object.target_files, {})
 
     # Test improperly formatted arguments.
     self.assertRaises(securesystemslib.exceptions.FormatError,
                       self.targets_object.remove_target, 3)
 
 
-    # Test invalid filepath argument (i.e., non-existent or invalid file.)
-    self.assertRaises(securesystemslib.exceptions.Error,
-                      self.targets_object.remove_target,
-                      '/non-existent.txt')
+    # # Test invalid filepath argument (i.e., non-existent or invalid file.)
+    # self.assertRaises(securesystemslib.exceptions.Error,
+    #                   self.targets_object.remove_target,
+    #                   '/non-existent.txt')
```

Comment:
There's no need to leave this commented out code, could you remove it?

Reply:
Squashed in https://github.com/theupdateframework/tuf/pull/1008/commits/3a13b7312a9148f7aac553affa371f3f26c69f11